### DISCUSSION TITLE
Viewing checks/status details for any commit

### DISCUSSION THREAD

**Discussion by mback2k** [community/community#3539] [Mobile]
At the moment the checks and status details can only be viewed for commits inside a PR, but I would really like to do that for any commit, e.g. on a branch.

With this bisecting issues on mobile would be a lot easier.

**Comment by candyho** [comment]
Thanks for sharing your feedback. I've added this request to our internal repo to keep track of similar requests and evaluate the priority from time to time. Hopefully we'll have an update to share with you soon.
